### PR TITLE
Word Embeddings updates based on comments

### PR DESCRIPTION
**Pull request by kendallc**

### CODE REVIEW

File: Word Embeddings/Word Embeddings.ipynb [hunk 1]
```diff
@@ -159,29 +173,34 @@
...
   },
   {
    "cell_type": "markdown",
    "metadata": {
     "slideshow": {
-     "slide_type": "slide"
+     "slide_type": "notes"
     }
    },
    "source": [
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: "zeroes" (a verb) should be "zeros" (noun).  In two places in first paragraph.

File: Word Embeddings/Word Embeddings.ipynb [hunk 2]
```diff
@@ -203,42 +233,68 @@
      "slide_type": "slide"
     }
    },
+   "outputs": [],
+   "source": [
+    "# Context is the number of words we are using as a context for the next word we want to predict\n",
+    "CONTEXT_SIZE = 2\n",
+    "# Embedding dimension is the size of the embedding\n",
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: Should you say, "size of the embedding **vector**" here?

File: Word Embeddings/Word Embeddings.ipynb [hunk 3]
```diff
@@ -203,42 +233,68 @@
...
-    "Where all the treasure of thy lusty days;\n",
-    "To say, within thine own deep sunken eyes,\n",
-    "Were an all-eating shame, and thriftless praise.\n",
-    "How much more praise deserv'd thy beauty's use,\n",
-    "If thou couldst answer 'This fair child of mine\n",
-    "Shall sum my count, and make my old excuse,'\n",
-    "Proving his beauty by succession thine!\n",
-    "This were to be new made when thou art old,\n",
-    "And see thy blood warm when thou feel'st it cold.\"\"\".split()\n",
     "# we should tokenize the input, but we will ignore that for now\n",
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: Is splitting the corpus not a form of tokenizing?  Perhaps define tokenize in this context.

File: Word Embeddings/Word Embeddings.ipynb [hunk 4]
```diff
@@ -354,7 +421,7 @@
   },
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: May want to remove this pip install of tqdm and we install from requirements.txt for workshop.

File: Word Embeddings/Word Embeddings.ipynb [hunk 5]
```diff
@@ -586,6 +697,28 @@
     "# "
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: We can now remove "Solutions in another directory" or say instead "Solutions will be found in the Solutions folder"

File: Word Embeddings/Word Embeddings.ipynb [hunk 6]
```diff
@@ -586,6 +697,28 @@
...
+    }
+   },
+   "source": [
+    "# Glossary\n",
+    "\n",
+    "* word embedding -- a dense vector representation of words\n",
+    "* one-hot encoding -- a sparse vector representation of words with ones\n",
+    "* vocabulary -- the set of words used in your language \n",
+    "* tokenization -- the process of breaking down bodies of text into words\n",
+    "* ReLU\n",
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: Do you want to add a definition for ReLU and NLL?

File: Word Embeddings/Word Embeddings.ipynb [hunk 7]
```diff
@@ -586,6 +697,28 @@
...
+    "* negative log likelihood\n",
+    "* loss function -- also known as a \"cost function\" a function to estimate the cost associated with an event\n",
+    "* Stochastic Gradient Descent (SGD) -- \"an iterative method for optimizing an objective function\" * (https://en.wikipedia.org/wiki/Stochastic_gradient_descent) \n",
+    "* learning rate -- a constant step to take in one iteration of stochastic gradient descnet\n",
+    "* autograd -- PyTorch method for performing backward propogation and gradient descent\n"
```
Comment (michhar) [inline Word Embeddings/Word Embeddings.ipynb]: `autograd` --  PyTorch's automatic differentiation class that performs the backpropagation gradient calculations automatically so that a "backward" class does not need to be defined by the programmer.

File: Word Embeddings/Word Embeddings.ipynb (hunk 5 above)
Comment (kendallcarta) [reply Word Embeddings/Word Embeddings.ipynb]: Okay, I'll remove the comment as not sure I'll have time to finish the solution before Friday!

File: Word Embeddings/Word Embeddings.ipynb (hunk 6 above)
Comment (kendallcarta) [reply Word Embeddings/Word Embeddings.ipynb]: Yeah, I was going to look for an image online
